tests/data/test_file.txt.md5

```diff
@@ -0,0 +1 @@
+169293f7c9138e4b50ebcab4358dc509
```

Comment:
> using MD5 to verify external binary data (which may have been initially provided by untrusted third parties) is specifically what it's weak to.

Not sure what you mean here. What weakness are you referring to that would not be present with a different hashing algorithm?